docs-source/content/samples/simple/azure-kubernetes-service/model-in-image.md

```diff
@@ -0,0 +1,836 @@
+    Desired State:  RUNNING
+    Health:
+      Activation Time:  2020-11-30T05:44:54.699Z
+      Overall Health:   ok
+      Subsystems:
+        Subsystem Name:  ServerRuntime
+        Symptoms:
+    Node Name:      aks-pool1model-71528953-vmss000000
+    Server Name:    managed-server1
+    State:          RUNNING
+    Cluster Name:   cluster-1
+    Desired State:  RUNNING
+    Health:
+      Activation Time:  2020-11-30T05:45:07.211Z
+      Overall Health:   ok
+      Subsystems:
+        Subsystem Name:  ServerRuntime
+        Symptoms:
+    Node Name:      aks-pool1model-71528953-vmss000001
+    Server Name:    managed-server2
+    State:          RUNNING
+    Cluster Name:   cluster-1
+    Desired State:  SHUTDOWN
+    Server Name:    managed-server3
+    Cluster Name:   cluster-1
+    Desired State:  SHUTDOWN
+    Server Name:    managed-server4
+    Cluster Name:   cluster-1
+    Desired State:  SHUTDOWN
+    Server Name:    managed-server5
+  Start Time:       2020-11-30T05:40:11.709Z
+Events:             <none>
+```
+{{% /expand %}}
+
+##### Access the application
+
+Access the Administration Console using the admin load balancer IP address, `http://52.191.234.149:7001/console`
+
+Access the sample application using the cluster load balancer IP address.
+
+```bash
+## Access the sample application using the cluster load balancer IP (52.191.235.71)
+$ curl http://52.191.235.71:8001/myapp_war/index.jsp
+
+<html><body><pre>
+*****************************************************************
+
+Hello World! This is version 'v1' of the mii-sample JSP web-app.
+
+Welcome to WebLogic Server 'managed-server1'!
+
+ domain UID  = 'sample-domain1'
+ domain name = 'domain1'
+
+Found 1 local cluster runtime:
+  Cluster 'cluster-1'
+
+Found 0 local data sources:
+
+*****************************************************************
+</pre></body></html>
+
+$ curl http://52.191.235.71:8001/myapp_war/index.jsp
+
+<html><body><pre>
+*****************************************************************
+
+Hello World! This is version 'v1' of the mii-sample JSP web-app.
+
+Welcome to WebLogic Server 'managed-server2'!
+
+ domain UID  = 'sample-domain1'
+ domain name = 'domain1'
+
+Found 1 local cluster runtime:
+  Cluster 'cluster-1'
+
+Found 0 local data sources:
+
+*****************************************************************
+</pre></body></html>
+```
+
+#### Rolling updates
+
+Naturally, you will want to deploy newer versions of the EAR application, located in the WDT archive ZIP file at `wlsdeploy/applications/myapp-v1`. To learn how to do this, follow the steps in [Update 3]({{< relref "/samples/simple/domains/model-in-image/update3" >}}).
+
+#### Clean up resources
+
+Run the following commands to clean up resources.
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/clean-up-resources-body-02.txt" >}}
+
+#### Troubleshooting
+
+  [Troubleshooting]({{< relref "/samples/simple/azure-kubernetes-service/troubleshooting.md" >}})
```

Comment:
[Troubleshooting] -> For troubleshooting advice (or information, whichever you prefer), see [Troubleshooting]... (period)